design/src/App.scss

```diff
@@ -77,6 +78,115 @@ $hover-color: #f8cc80;
     }
 }
 
+.glogg-h1 {
+    font-family: 'Merienda', cursive;
+    font-size: 35pt;
```

Comment:
Font sizes should be set in `rem`/`em` for accessibility. `pt ` is usually used in printing :)

As you're using Sass, add a function for converting px or pt values to rem.

For example:
```
@function pxToRem($px) { @return ($px / 16px) * 1rem; }
font-size: pxToRem(40px);
```

Reply:
har fixat det